src/components/Accounts/Actions/CancellationModal.js

```diff
@@ -17,7 +17,7 @@ import {
 } from '@folio/stripes/components';
 
 import css from './modal.css';
-import { localizeCurrencyAmount } from '../../util/localizeCurrencyAmount';
+import localizeCurrencyAmount from '../../util/localizeCurrencyAmount';
```

Comment:
after updating src/hooks/index.js file on the lines of other hooks exports (as suggested below in one of the comments), adjust this import